crates/rnote-ui/po/rnote.pot

```diff
@@ -2318,3 +2332,114 @@ msgstr ""
 #: crates/rnote-ui/src/workspacebrowser/workspaceactions/createfolder.rs:75
 msgid "New Folder"
 msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:144
+msgctxt "used in string representation of the current selected color"
+msgid "grey"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:145 crates/rnote-ui/src/utils.rs:157
+msgctxt "part of string representation of a color"
+msgid "rose"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:146
+msgctxt "part of string representation of a color"
+msgid "red"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:147
+msgctxt "part of string representation of a color"
+msgid "orange"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:148
+msgctxt "part of string representation of a color"
+msgid "yellow"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:149
+msgctxt "part of string representation of a color"
+msgid "chartreuse-green"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:150
+msgctxt "part of string representation of a color"
+msgid "green"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:151
+msgctxt "part of string representation of a color"
+msgid "spring-green"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:152
+msgctxt "part of string representation of a color"
+msgid "cyan"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:153
+msgctxt "part of string representation of a color"
+msgid "azure"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:154
+msgctxt "part of string representation of a color"
+msgid "blue"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:155
+msgctxt "part of string representation of a color"
+msgid "violet"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:156
+msgctxt "part of string representation of a color"
+msgid "magenta"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:158 crates/rnote-ui/src/utils.rs:164
+#: crates/rnote-ui/src/utils.rs:170
+msgctxt "part of string representation of a color"
+msgid "invalid"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:161
+msgctxt "part of string representation of a color"
+msgid "desaturated"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:163
+msgctxt "part of string representation of a color"
+msgid "vibrant"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:167
+msgctxt "part of string representation of a color"
+msgid "dark"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:168
+msgctxt "part of string representation of a color"
+msgid "mid"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:169
+msgctxt "part of string representation of a color"
+msgid "bright"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:174
+msgctxt "part of string representation of a color"
+msgid "transparent"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:176
+msgctxt "part of string representation of a color"
+msgid "black"
+msgstr ""
+
+#: crates/rnote-ui/src/utils.rs:178
+msgctxt "part of string representation of a color"
+msgid "white"
+msgstr ""
```

Comment:
The rule for `gettext` to work correctly is to match the line the `pgettext` is called in the `pot` file right ?
I'm not 100 % sure it's the case here

line number in `utils.rs` vs `rnote.pot` (and associated name)

148/145/rose
150/146/red
153/147/orange
156/148/yellow
158/149/chartreuse-green
163/150/green
166/151/spring-green
169/152/cyan
172/153/azure
175/154/blue
178/155/violet
181/156/magenta
183/157/rose
184/158/invalid

187/161/desaturated
189/163/vibrant
190/164/invalid

193/167/dark
195/168/mid
197/169/bright
198/170/invalid

202/174/transparent
204/176/black
206/178/white

Reply:
I *think* the line number only helps for matching or even is only a help for translators to get additional context by looking at the source code, the msgid is what will be matched against in the end.
I don't know why it wrote the wrong line numbers, but it seems to be fixed with the latest execution of `meson compile rnote-pot` (though I only checked a few entries)